### PR TITLE
APP-266: Fix "Zeppelin data analytics demo" documentation

### PR DESCRIPTION
**Pull request by vasner**
Fix "Zeppelin data analytics demo" documentation

Reviewers: @Slash32 and @Rib47

### CODE REVIEW

File: zeppelindemo/resources/projects.xml [hunk 1]
```diff
@@ -54,16 +54,19 @@ documents with SQL, Scala and virtually any other language or data-processing-ba
...
+3. Download Apache Zeppelin and install it to your Sandbox.
+Execute following commands:
 <pre>
 $ cd /opt
-$ sudo wget http://www.eu.apache.org/dist/incubator/zeppelin/0.5.6-incubating/zeppelin-0.5.6-incubating-bin-all.tgz
-$ sudo tar -xzvf zeppelin-0.5.6-incubating-bin-all.tgz
-$ sudo ln -s /opt/zeppelin-0.5.6-incubating-bin-all /opt/zeppelin
+$ sudo wget http://archive.apache.org/dist/zeppelin/zeppelin-0.6.1/zeppelin-0.6.1-bin-all.tgz
+$ sudo tar -xzvf zeppelin-0.6.1-bin-all.tgz
+$ sudo ln -s /opt/zeppelin-0.6.1-bin-all.tgz /opt/zeppelin
```
Comment (Rib47) [inline zeppelindemo/resources/projects.xml]: No ".tgz" needed. This is mistake in ticket, I have correct it.
**sudo ln -s /opt/zeppelin-0.6.1-bin-all /opt/zeppelin**

Comment (vasner) [reply zeppelindemo/resources/projects.xml]: Fixed.

Comment (Rib47) [reply zeppelindemo/resources/projects.xml]: Looks good now

**Comment by Slash32** [comment]
LGTM

**Pull request merged**